src/commands/balance.ts

```diff
@@ -0,0 +1,33 @@
+import { CommandInteraction } from 'discord.js';
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { Category } from 'types/applicationCommand';
+export async function run(
+    client: FluorineClient,
+    interaction: CommandInteraction
+) {
+    const balance = await client.economy.get(
+        interaction.user.id,
+        interaction.guild.id
```

Comment:
```suggestion
        interaction.guildId
```
`interaction.guild.id` can throw errors sometimes, `guildId` is guaranteed to always work